src/shared/components/common/combobox/Combobox.tsx

```
@@ -0,0 +1,153 @@
+  );
+}
+
+export function ComboboxTrigger() {
+  const ctx = React.useContext(ComboboxContext);
+  if (!ctx) return null;
+  const { open, setOpen, options, value, placeholder } = ctx;
+  return (
+    <PopoverTrigger asChild>
+      <Button
+        variant='secondary'
+        role='combobox'
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)